localstack-core/localstack/utils/docker_utils.py

```diff
@@ -154,16 +154,16 @@ def container_ports_can_be_bound(
             remove=True,
         )
     except Exception as e:
-        if "port is already allocated" not in str(e) and "address already in use" not in str(e):
+        if "address already in use" not in str(e):
             LOG.warning(
-                "Unexpected error when attempting to determine container port status: %s", e
+                "Unexpected error when attempting to determine container port status", exc_info=e
```

Comment:
```suggestion
                "Unexpected error when attempting to determine container port status: e", exc_info=LOG.isEnabledFor(logging.DEBUG)
```
This will only print the stack trace when debug logging is enabled, not for every call, which would reduce the logging in higher log levels. WDYT?

Reply:
If it was a regular scenario, then yes, it makes sense to only show this on Debug. But this is an unhandled exception, and something happened here that we didn't expect (or know how to handle). So IMO showing a stacktrace at any log level makes it clearer to the user that this is an actual error.

Reply:
Valid point, fine for me!